.travis.yml

```diff
@@ -1,59 +1,54 @@
 language: swift
-os: osx
-osx_image: xcode10.2
+
+env:
+  global:
+    - IOS_SCHEME="Nuke-WebP-Plugin iOS"
+    - MACOS_SCHEME="Nuke-WebP-Plugin macOS"
+    - TVOS_SCHEME="Nuke-WebP-Plugin tvOS"
+    - WATCHOS_SCHEME="Nuke-WebP-Plugin watchOS"
+  matrix:
```

Comment:
Hey, I recently updated the approach for manning .travis.yml with separate jobs instead of the matrix. I think it's much cleaner, I suggest to try this https://github.com/kean/Nuke/blob/master/.travis.yml

Reply:
@kean
I saw it!
I think it's enviable because it's written very neatly.
Since the description of .travis.yaml was complicated and the same test was executed with both push and pull_request, I am trying to clean it.

However, Travis CI is still slow, so I am thinking about moving to Github Actions!

Reply:
Yeah, GitHub Actions might be even better. I'm using it for new projects. I'm not sure it can do everything Travis can do, I'm going to try migrating Nuke to it later too.

Reply:
I recently touched GitHub Actions at work and it's so fast that I really like it.
It will be a big change to do in this PR, so I plan to do it in another PR.

Reply:
Fast is an advantage. Every time I try to change something in Travis config, it doesn't work the first time, and I end up spending hours iterating on it 😭

Reply:
I also made a change so that it only works with pull_request, and env etc was destroyed, so I'm tired of exhausting it with Travis CI 😢